### PR TITLE
 added ORA to oracles folder in docs

### PR DESCRIPTION
**Pull request by Gwen-M**
ORA OAO is deployed on Linea and available since a few months now, so here's a quickstart and few infos to help devs using OAO on Linea.

I tested it and it runs locally.

### CODE REVIEW

File: docs/developers/tooling/oracles/ora.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+[ORA](https://ora.io) is Ethereum's Trustless AI. 
+As a verifiable oracle protocol, ORA brings AI and complex compute onchain.
+Its main product, **Onchain AI Oracle (OAO)**, integrates AI capabilities directly onchain.
+
+ORA breaks down the limitations of smart contracts by offering verifiable AI inference, so developers can innovate freely.
+
+## OAO Quickstart
+
+This quickstart is designed to help you to build a smart-contract on Linea able to interact with OAO. You can find more details in [our docs Quickstart](https://docs.ora.io/doc/oao-onchain-ai-oracle/develop-guide).
```
Comment (jlwllmr) [inline docs/developers/tooling/oracles/ora.md:14]:
```suggestion
This quickstart is designed to help you to build a smart contract on Linea able to interact with OAO. You can find more details in [our docs Quickstart](https://docs.ora.io/doc/oao-onchain-ai-oracle/develop-guide).
```

File: docs/developers/tooling/oracles/ora.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+3. When you want to initiate an AI inference request, call OAO as follows:
+
+```solidity
+aiOracle.requestCallback(modelId, input, address(this), gas_limit, callbackData);
+```
+
+## Reference
+
+**2 models** are available on Linea: Stable Diffusion (ID: 50) and Llama3 8B Instruct (ID: 11).
```
Comment (jlwllmr) [inline docs/developers/tooling/oracles/ora.md:57]:
```suggestion
**Two models** are available on Linea: Stable Diffusion (ID: 50) and Llama3 8B Instruct (ID: 11).
```

File: docs/developers/tooling/oracles/ora.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+[Prompt](https://docs.ora.io/doc/oao-onchain-ai-oracle/reference) and [SimplePrompt](https://docs.ora.io/doc/oao-onchain-ai-oracle/reference) are both example smart contracts interacted with OAO.
+
+For simpler application scenarios (eg. Prompt Engineering based AI like GPTs), you can directly use Prompt or SimplePrompt.
```
Comment (jlwllmr) [inline docs/developers/tooling/oracles/ora.md:61]:
```suggestion
For simpler application scenarios (eg. Prompt Engineering-based AI like GPTs), you can directly use Prompt or SimplePrompt.
```

**Review by jlwllmr — CHANGES_REQUESTED** [review]
Thanks for putting this together -- just some minor changes needed.

**Comment by Gwen-M** [comment]
I've made the suggested changes, thanks :)

**Review by jlwllmr — APPROVED** [review]

**Pull request merged**